docs/reference/project/buckets.mdx

```diff
@@ -1 +1,16 @@
-TODO...
+A nitric bucket is described by its definition in a `nitric.yaml` file. A bucket represents a container for the storage of blobs/binary data.
+
+## Properties
+
+> Buckets do not have properties at present and are reprensented only by their name at this time this is subject to change over time as requirements around bucket configuration emerge
```

Comment:
reprensented -> represented.
It should probably be a new sentence after 'at this time', otherwise it reads awkwardly.

Reply:
fixed